modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/openy_prgf_event_listing.info.yml

```diff
@@ -0,0 +1,18 @@
+name: 'OpenY Paragraph Event Posts Listing'
+description: 'OpenY Paragraph Event Posts Listing.'
+type: module
+core: 8.x
+package: 'YMCA Maryland'
```

Comment:
provide a version, please

Reply:
@dbuzinov please change package to "OpenY"